oauth2/oauth2_auth_code_test.go

```diff
@@ -392,6 +392,41 @@ func TestAuthCodeWithDefaultStrategy(t *testing.T) {
 								_, err := oauthConfig.TokenSource(oauth2.NoContext, token).Token()
 								require.Error(t, err)
 							},
+						}, {
+							d:       "Checks if request fails when subject doesn't match",
```

Comment:
The test is actually that this fails if the subject is empty, right?

Reply:
Yup, you're right!